sdk/appcenter/src/main/java/com/microsoft/appcenter/Flags.java

```diff
@@ -24,4 +28,22 @@
      * Default combination of flags.
      */
     public static final int DEFAULT_FLAGS = PERSISTENCE_NORMAL;
```

Comment:
Isn't it redundant to call this `DEFAULT_FLAGS`?  Why not just 'FLAGS`?

Reply:
Android uses either DEFAULT_FLAGS in Notification or lowercase defaults (with an s) in Notification but is a deprecated field. That's where the choice comes from.